contracts/contracts/core/ProviderRegistry.sol

```diff
@@ -127,6 +129,7 @@ contract ProviderRegistry is
         }
 
         emit FundsSlashed(provider, residualAmt + penaltyFee);
+        emit FundsSlashedV2(provider, residualAmt + penaltyFee, commitmentDigest);
```

Comment:
Why not just add a field to the existing `FundsSlashed` event?

Reply:
To maintain backwards compatibility

Reply:
There is a reference to this event in the dashboard. We will need to update that as well.

https://github.com/primev/mev-commit/blob/main/tools/dashboard/stathandler.go#L270